url/url-tojson.any.js

```diff
@@ -0,0 +1,4 @@
+test(() => {
+  const a = new URL('https://example.com/');
+  assert_equals(JSON.stringify(a), '"https://example.com/"');
```

Comment:
Could you undo the quoting changes to see if GitHub then recognizes this as a rename + modifications rather than a new resource?

Reply:
@mariestaver Undid the quoting changes, but this is still being interpreted as a new file